workbench-core/environments/src/postDeployment/authorizationSetup.ts

```diff
@@ -57,15 +81,44 @@ export default class AuthorizationSetup {
     }
   }
 
+  /**
+   * Assigns user to group
+   * @param userId - Group description
+   * @param groupName - Name of the Group to assign
+   */
+  public async assignUserToGroup(userId: string, groupName: string, user: AuthenticatedUser): Promise<void> {
+    console.log(`Assigning user ${userId} to ${groupName} group.`);
+    await this._authService.addUserToGroup({
+      groupId: groupName,
+      userId,
+      authenticatedUser: user
+    });
+
+    console.log(`Assigned user ${userId} to ${groupName} group.`);
+  }
+
   /**
    * Creates set of permissions
    * @param permissions - Permissions to create
    */
-  public async createIdentityPermissions(permissions: IdentityPermission[]): Promise<void> {
+  public async createIdentityPermissions(
+    permissions: IdentityPermission[],
+    user: AuthenticatedUser
+  ): Promise<void> {
     console.log(`Creating identity permissions.`);
-    await this._authService.createIdentityPermissions({
-      identityPermissions: permissions
-    });
+    try {
+      await this._authService.createIdentityPermissions({
+        identityPermissions: permissions,
+        authenticatedUser: user
+      });
+    } catch (err) {
+      if (isIdentityPermissionCreationError(err)) {
+        console.warn(err);
```

Comment:
Works for now but we should have a post-deployment report that will let you know if something went wrong.

Reply:
Unfortunately currently we don't have a good indication on whether everything worked fine or not. The error from DDB transaction is not propagated properly, so normal situation "permissions already exist" cannot be distinguished from "something went wrong". Screenshot for reference:
![image](https://user-images.githubusercontent.com/112517151/214863766-f8a6a0d3-f775-4861-966a-8b2b4e7420eb.png)

tagging @htztomic / @bhudnell to see if it's possible to provide better error message